packages/runtime/src/runtime/modal.tsx

```diff
@@ -164,6 +207,7 @@ export const ModalWrapper: React.FC<PropsWithChildren> = ({ children }) => {
             ? getPositionSelector(
                 modalDimensions[index].width,
                 modalDimensions[index].height,
+                options.height?.includes("100"),
```

Comment:
I don't think you even need the inline style selector you can pass styles directly to the modal to position. https://codesandbox.io/p/sandbox/to-customize-the-position-of-modal-antd-5-14-2-23xtnp?file=%2Fdemo.tsx%3A15%2C28